### PR TITLE
test script for check_stop_trading_abci

### PR DESCRIPTION
**Pull request by Ravleen-Solulab**
This is branch for testing script for rounds and dialogues.

### CODE REVIEW

**Review by Adamantios — CHANGES_REQUESTED** [review]
Could you also add a test for the `SynchronizedData`?

**Comment by Ravleen-Solulab** [comment]
Sure, no problem. I will do it.

**Review by Adamantios — CHANGES_REQUESTED** [review]
There is uncovered code:

- In check stop trading: synced data
- In staking: synced data and end block in CallCheckpointRound
- In multiplexer: end block in PostTxSettlementRound

**Comment by Ravleen-Solulab** [comment]
1. In check_stop_trading_abci: added test_synchronize_data_initialization
line 128.
2.  For staking added end block from line 91. Will change the name of
test_call_checkpoint_round as I implement end block in it.
3. Same for tx settlement ,  from line 96 added end block in
test_post_tx_settlement_round_end_block.

Let me know if I am missing something.
Thank you.

File: packages/valory/skills/check_stop_trading_abci/tests/test_rounds.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    synchronized_data = MagicMock()
+    logger = MagicMock()
+    context = MagicMock()
+
+    # Instantiate CheckStopTradingAbciApp with mocked parameters
+    return CheckStopTradingAbciApp(synchronized_data=synchronized_data, logger=logger, context=context)
+
+
+
+def test_check_stop_trading_round_initialization(synchronized_data):
```
Comment (dvilelaf) [inline packages/valory/skills/check_stop_trading_abci/tests/test_rounds.py:72]: These tests should also check whether we can run the rounds and that for every potential execution branch  we get the expected data and event. Check this for some inspiration: https://github.com/valory-xyz/IEKit/blob/main/packages/valory/skills/dynamic_nft_abci/tests/test_rounds.py#L125

**Comment by Ravleen-Solulab** [comment]
Thanks, I'll check it out and make changes accordingly.

File: packages/valory/skills/market_manager_abci/tests/test_rounds.py [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        )
+
+ 
+
+class TestUpdateBetsRound(BaseMarketManagerRoundTestClass):
+    """Test UpdateBetsRound execution."""
+
+    def test_update_bets_round_execution(self):
+        """Test the execution of UpdateBetsRound."""
+        test_case = {
```
Comment (dvilelaf) [inline packages/valory/skills/market_manager_abci/tests/test_rounds.py:110]: Ideally these test cases are parametrized. See here: https://github.com/valory-xyz/IEKit/blob/main/packages/valory/skills/dynamic_nft_abci/tests/test_rounds.py#L160

**Pull request closed without merging**